src/ScottPlot4/ScottPlot/SnapLogic/Custom.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace ScottPlot.SnapLogic;
```

Comment:
`SnapLogic` is kind of a weird namespace? It's source code, everything is logic anyways?

Perhaps `ScottPlot.Snap` or `ScottPlot.Snapping`?

Reply:
I like `Snapping`, and further I like `ScottPlot.Snapping.1D` and `ScottPlot.Snapping.2D`